test/fixtures/mson-mixin-nonexistent.apib

```diff
@@ -9,4 +9,4 @@
 
 ## User (object)
 - id
-- Include Address
+- Include XXX
```

Comment:
When snowcrash returns an error, the AST it gives should be neglected. The AST should not even be tested when the parser has an error. This is why, the above 2 samples have different AST, because we are returning an error as soon as we see one.